serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -0,0 +1,101 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-jwt"]
+= Using JSON Web Token authentication with {ProductShortName} and {ServerlessProductName}
+:context: serverless-ossm-jwt
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This section provides information about how you can enable JSON Web Token (JWT) authentication for Knative services.
```

Comment:
I think this can be simplified to be more minimalistic, but feel free to disregard:

> You can enable JSON Web Token (JWT) authentication for Knative services.

Reply:
+1